### ISSUE TITLE
No keyboard access to expandable section at bottom of sidebar

### ISSUE THREAD

**Issue by ChrisXKeroack** [intel/openfl#843]
**Describe the bug**
A keyboard user cannot access the versioning links in the bottom of the sidebar.

**To Reproduce**
1. https://openfl.readthedocs.io/en/latest/index.html 
2. Press TAB repeatedly to navigate between interactive elements.

Notice: Keyboard focus skips the versioning links section and goes from sidebar to main content.

**Expected behavior**
Ability to put keyboard focus on versioning section.

**Screenshots**
![image](https://github.com/securefederatedai/openfl/assets/104644211/a416b8b8-1d54-4d3a-a3fb-75decfff15a8)

**Desktop:**
This issue is not OS or browser specific, but for tracking purposes, testing was done on Win11, in Edge Version 114.0.1823.58 (Official build) (64-bit).

**Smartphone:**
This issue is not device, OS, or browser specific.

**Additional context**
This is an accessibility issue, please annotate this bug with the 'accessibility' label.

**Comment by fstrr** [comment]
This is an issue with the [Read The Docs platform](https://github.com/readthedocs/readthedocs.org).

**Comment by theakshaypant** [comment]
Closing. As mentioned by @fstrr, this is an issue with [Read The Docs platform](https://github.com/readthedocs/readthedocs.org).
